### PR TITLE
Animation designer prototype

### PR DESCRIPTION
**Pull request by ngzhian**
Uses SnapKit, please `pod install`.
Everything is hardcoded, full of magic constants, but at least there's something out there.
The issue that tracks my work on the animation designer is #42.

screen shot 2017-03-30 at 10 32 28 am

I created 2 custom views `TimelineView` and `ColourPicker`, their name suggests what they are for. They look the same now because I'm just drawing squares, when then design is ready I'll change how they look.
The segmented control is for switching between absolute and relative animation design, related to #35 .

### CODE REVIEW

File: MIDIchlorians/MIDIchlorians/TimelineView.swift [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+class TimelineView: UIView {
+    // a data structure of animation frames
+    // keep track of which frame is selected currently and show it differently
+    private let fullCircle: CGFloat = CGFloat(2 * M_PI)
+    private var selectedFrameIndex: Int?
+    let frames: [Bool] = [
+        true, false,
+        true, false,
+        true, false,
```
Comment (advaypal) [inline MIDIchlorians/MIDIchlorians/TimelineView.swift:19]: Boolean values are to indicate if at least one gird has been assigned a colour in this frame?

**Comment by advaypal** [comment]
Nice! LGTM

**Review by advaypal — APPROVED** [review]

File: MIDIchlorians/MIDIchlorians/TimelineView.swift (hunk 1 above)
Comment (ngzhian) [reply MIDIchlorians/MIDIchlorians/TimelineView.swift:19]: yup :)

Comment (ngzhian) [reply MIDIchlorians/MIDIchlorians/TimelineView.swift:19]: at least for now lol

**Comment by anand-sundaram** [comment]
@ngzhian LGTM (apart from the hardcoding/magic constant issues you've already addressed). Do you intend to merge this already, or is this PR wip?

**Comment by ngzhian** [comment]
I wanna merge this and continue working on this. And also maybe over the weekend yall can help out too :)

**Pull request merged**